inference-engine/thirdparty/clDNN/src/graph_optimizer/prepare_quantization.cpp

```diff
@@ -24,8 +26,8 @@ template<typename T>
 bool check_binarization(memory::ptr mem_input_low, memory::ptr mem_input_high, program& p) {
     bool is_binarization = true;
     const auto& stream = p.get_stream();
-    mem_lock<T> data_input_low_lock{mem_input_low, stream};
-    mem_lock<T> data_input_high_lock{mem_input_high, stream};
+    mem_lock<T/*, mem_lock_type::read*/> data_input_low_lock{mem_input_low, stream};
```

Comment:
New mem_lock_type is now merged. We can uncomment it?

Reply:
Yes, of course. I'll update it soon.

Reply:
Fixed